### ISSUE TITLE
[BUG] Homebrew cleanup not working as expected. 

### ISSUE THREAD

**Issue by uzza1hossain** [alienator88/Pearcleaner#63]
**Describe the bug:** <br>

**Steps to reproduce:**
1. Install a cask via homebrew (I just tested with only-switch)
2. Uninstall using Pearcleaner.
3. Run `brew info package-name`
4. It's still shows installed and I have to uninstall it manually.

**Expected behavior** <br>
Automatically uninstall from homebrew
 
**Info:**
 - OS: [14.5]
 - Pearcleaner Version: [ 3.6.5]

**Comment by alienator88** [comment]
I found the issue..a couple reasons it didn't work:
1. It wasn't handling spaces as dashes in the names, so fixed that first.
2. Unrelated to `only-switch`, but other casks have identifiers that don't match the app name, so I had to add some checks in there first to look at the brew list of casks and match them up with the app name being uninstalled

I'll have this fix out in the next version. Waiting to get another feature request completed before I release this one.
